brands/amenity/fast_food.json

```diff
@@ -2114,6 +2114,15 @@
       "takeaway": "yes"
     }
   },
+  "amenity/fast_food|Texas Burger": {
+    "tags": {
+      "amenity": "fast_food",
+      "brand": "Texas Burger",
+      "cuisine": "burger",
+      "name": "Texas Burger",
+      "takeaway": "yes"
+    }
+  }
```

Comment:
I think you're missing a comma after this bracket

Reply:
Thank you. It's taken care of now.